acl.go

```diff
@@ -0,0 +1,50 @@
+package dtrack
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/google/uuid"
+)
+
+type AclService struct {
```

Comment:
Please use uppercase for abbreviation so casing is consistent (like you did for LDAP).

```suggestion
type ACLService struct {
```